src/aeso_stdlib.erl

```diff
@@ -0,0 +1,199 @@
+%%%-------------------------------------------------------------------
+%%% @author Radosław Rowicki
+%%% @copyright (C) 2019, Aeternity Anstalt
+%%% @doc
+%%%     Standard library for Sophia
+%%% @end
+%%% Created : 6 July 2019
+%%%
+%%%-------------------------------------------------------------------
+
+-module(aeso_stdlib).
+
+-export([stdlib/0]).
+
+stdlib() ->
+    maps:from_list(
+      [ {<<"List.aes">>, std_list()}
+     %% , {<<"Func.aes">>, std_function()}
+      ]
+     ).
+
+std_function() ->
+"
+namespace Func =
+
+  function id(x) = x
+
+  function const(x) = (y) => x
+
+  function comp(f, g) = (x) => f(g(x))
+
+  function iter(n, f) = iter_(n, f, (x) => x)
+  private function iter_(n, f, acc) =
+    if(n == 0) acc
+    elif(n == 1) comp(f, acc)
+    else iter_(n / 2, f, if(n mod 2 == 0) comp(acc, acc) else comp(f, comp(acc, acc)))
```

Comment:
This isn't right. `iter(2 ^ n, f) == f`. You want the recursive call to be
```suggestion
    else iter_(n / 2, comp(f, f), if(n mod 2 == 0) acc else comp(f, acc))
```
However I don't think this is really a function that we should put in the standard libarary.

Reply:
My bad, sorry, fixed. I believe it can be used in some iterative algorithms and we don't gain much if we remove it